pro/src/new_components/OfferIndividualForm/OptionDuo/OptionDuo.tsx

```diff
@@ -0,0 +1,31 @@
+import { useFormikContext } from 'formik'
+import React from 'react'
+
+import FormLayout from 'new_components/FormLayout'
+import { Checkbox } from 'ui-kit'
+
+import { IOfferIndividualFormValues } from '../types'
+
+const OptionDuo = (): JSX.Element | null => {
+  const {
+    values: { subCategoryFields },
+  } = useFormikContext<IOfferIndividualFormValues>()
+
+  if (!subCategoryFields.includes('isDuo')) {
+    return null
+  }
+
+  return (
+    <FormLayout.Section title='Réservations "Duo"'>
```

Comment:
```suggestion
    <FormLayout.Section title='Réservations “Duo”'>
```
typo (changement de guillemets)